swdc

```diff
@@ -66,13 +66,43 @@ function applyFixture()
   docker-compose run --rm cli mysql -h mysql -u root -proot $SHOPWARE_PROJECT < "${DIR}/fixtures/${FIXTURE_NAME}.sql"
 }
 
+function isComposerProject()
+{
+    if [ -f "$SHOPWARE_FOLDER/app/install.sh" ]; then
+        echo "true"
+    fi
+}
+
+function buildComposerProjectEnvFile()
+{
+  echo 'SHOPWARE_ENV="development"' > "$SHOPWARE_FOLDER/.env"
+  # TODO do better than this
+  echo 'SHOPWARE_VERSION="5.5.4"' >> "$SHOPWARE_FOLDER/.env"
```

Comment:
That looks very wrong to me
They are not required anymore in the newst composer project version

Reply:
From where is the shopware retrieved otherwise? Do I need to care?

Reply:
Shopware parses it from installed version in composer https://github.com/shopware/composer-project/blob/master/app/AppKernel.php#L64

Reply:
Nice. Todo solved.